harness/determined/pytorch/_pytorch_context.py

```diff
@@ -800,6 +840,31 @@ def current_train_batch(self) -> int:
             raise det.errors.InternalException("Training hasn't started.")
         return self._current_batch_idx
 
+    @staticmethod
+    def _add_prefix_in_state_dict_if_not_present(state_dict: Dict[str, Any], prefix: str) -> None:
```

Comment:
is this coming from somewhere or is it newly added?

Reply:
my bad. i had moved it here to support checkpoint loading.